src/photos/styles/toolbar.styl

```diff
@@ -29,6 +29,12 @@
     margin-left auto
 
     a
+        &.pho-action-upload
+            padding-left 1rem
+
+            svg
+                margin-right .5rem
+                vertical-align text-top
```

Comment:
Are you sure about the `text-top` align? Maybe it's because I'm paying attention, but it looks weird to me : 
<img width="253" alt="capture d ecran 2018-01-09 11 00 54" src="https://user-images.githubusercontent.com/2261445/34715314-733bffe6-f52c-11e7-8c5e-dc337bbe51ce.png">

Reply:
It's not centered also without it. This bit needs a good ol' refactoring but… 
Or maybe I could try something else… wait.